### PR TITLE
Update error and help alert UI

### PR DESCRIPTION
**Pull request by line47**
Part of the UI/UX work that @bpdesigns and I are tackling. 

This PR updates all error messages to use the USWDS alert error/help styling 
https://designsystem.digital.gov/components/alerts/

- There was inconsistency with errors  classes/HTML (this makes them consistent!) 🎉
- This PR also moves the error from being after the field (which felt really awkward) to between the label and text input. Now the help and error alerts show in the same spot

tested on MacOS
- Chrome 67.0.3396.99
- Firefox 55.0.2
- Safari 11.1.2

Tested on Windows
- Edge
- IE 11

Screenshots abound!

---
# new alerts
screen shot 2018-08-15 at 8 30 53 am
screen shot 2018-08-15 at 8 32 25 am
screen shot 2018-08-15 at 8 32 35 am
screen shot 2018-08-15 at 8 46 59 am
screen shot 2018-08-15 at 9 53 50 am

# old alerts
screen shot 2018-08-15 at 8 35 43 am
screen shot 2018-08-15 at 8 33 48 am
screen shot 2018-08-15 at 8 34 46 am
screen shot 2018-08-15 at 8 47 09 am
screen shot 2018-08-15 at 9 53 42 am

### CODE REVIEW

File: src/components/ErrorList/ErrorList.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class ErrorList extends React.Component {
       })
       sectionErrors.push(
         <span key={title}>
-          <h3>{title}</h3>
+          <h4>{title}</h4>
```
Comment (line47) [inline src/components/ErrorList/ErrorList.jsx]: Made an `H4` as the H tag heading for the error list is an `H3`

**Comment by ryanhofdotgov** [comment]
@line47 Can you note what browsers you've tested these changes under, and perhaps the rest of us can fill in the gaps as we review? It looks like there might be enough changes here to warrant a higher level of testing than for a smaller tweak.

**Comment by bpdesigns** [comment]
LGTM

**Comment by line47** [comment]
@ryanhofdotgov I checked in MacOS so far and all looks consistent.

- Chrome 67.0.3396.99
- Firefox 55.0.2
- Safari 11.1.2

Having a time getting Virtual box and Docker to run at the same time and not grind my laptop to a halt. 
Maybe you have a quicker way to test the Windows cases?

**Comment by ryanhofdotgov** [comment]
@line47 I can run it through IE11 and Edge when I review it, but might be another day or so until I get to that.

**Comment by ryanhofdotgov** [comment]
Also: we can ignore patch test failures for time being – need to figure out how to use these reports more effectively.

**Comment by line47** [comment]
@ryanhofdotgov Awesome, thanks and no worries!

**Comment by ryanhofdotgov** [comment]
Going to need to defer my IE testing until next week.

**Comment by line47** [comment]
Tested in IE11 and Edge

**IE11 Screesnhots look good!**
ie-11-1
ie-11-2
ie-11-3
ie-11-4
ie-11-5
ie-11-6
edge-6

**Edge Screenshots**
edge-1
edge-2
edge-3
edge-4
edge-5

**Comment by line47** [comment]
@ryanhofdotgov and @hursey013 this is ready for review - tested in Windows IE 11 and Edge

**Pull request merged**